assets/payments.css

```
@@ -0,0 +1,26 @@
+.payments__p {
+  height: 30px;
+}
+
+.payments__list {
+  margin-top: 8px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 30px;
+}
+.payments__item {
+}
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)